Tests/BootCamp.Chapter.Tests/GameComponents/CardsTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BootCamp.Chapter.Gambling;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests.GameComponents
+{
+    public class CardsTests
+    {
+        [Theory]
+        [MemberData(nameof(GetCardAndName))]
+        public void ToString_Returns_Expected_String(Card card, string expectedName)
```

Comment:
I think this can be a fact and just check if one card works. There is nothing different in all other cards.